apps/modernization-ui/src/providers/UserContext.tsx

```diff
@@ -51,7 +51,7 @@ const UserContextProvider = ({ initial, children }: Props) => {
             document.cookie = 'nbs_token=; Max-Age=0; path=/;';
             if (Config.enableLogin) {
                 // loading external page will clear state
-                window.location.href = '/nbs/logOut';
+                window.location.href = '/logout';
```

Comment:
issue:  This should `/nbs/logout` all lowercase because that is the route in NBS 6 that will actually log the user out.  The `/nbs/logOut` is the page that displays the message after the user has actually been logged out.